### DISCUSSION TITLE
installing dune-configurator

### DISCUSSION THREAD

**Discussion by progman1** [ocaml/dune#8508] [General]
from a clone of the dune repo how should a single package be installed?
I used 
```
dune build  -p dune-configurator @install
dune install -p dune-configurator
```
and got this:
```
Error: The following <package>.install are missing:
- _build/default/dune.install
- _build/default/dune-action-plugin.install
- _build/default/dune-glob.install
- _build/default/dune-private-libs.install
- _build/default/dune-rpc.install
- _build/default/dune-rpc-lwt.install
- _build/default/dune-site.install
- _build/default/dyn.install
- _build/default/fiber.install
- _build/default/ocamlc-loc.install
- _build/default/stdune.install
Hint: try running: dune build [-p <pkg>] @install
```
I can build most of those dependencies by listing them for -p but
when getting to dune-rpc-lwt I cannot as I don't have lwt installed.
since opam did in fact install dune/dune-configurator in another switch successfully
which didn't have lwt,  indicates that not all these dependencies are actually required.

since the above hint is exactly as I already did, is there a problem here, a special situation?

**Comment by Alizter** [comment]
You want to do `dune build -p dune-configurator` and then `dune install dune-configurator`. (dune install doesn't have a -p option).

If `dune build -p dune-configurator` complains about missing dependencies, then build and install those first.
